config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```diff
@@ -139,6 +139,54 @@ fields:
             default:
               region: content
 
+    -   name: exp_dataset
+        content_type: research_experiment
+        label: Datasets
+        type: chado_property_type_default
+        description: Use this field to describe specific datasets to be attached to the experiment. You should preferentially use the importers for data. This field is meant for datasets not yet supported by our system. Make sure to describe every column of tabular data.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: NCIT
+            termAccession: C25474
+        display:
+          view:
+            default:
+              region: content
+              label: hidden
+          form:
+            default:
+              region: content
+
+    -   name: exp_research_outputs
+        content_type: research_experiment
+        label: Research Outputs
+        type: chado_property_type_default
+        description: Include non-dataset focused research outputs here such as software and presentations.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: CO_010
+            termAccession: '0000159'
```

Comment:
I did a really bad job choosing a term here 🙈 I must have meant to change it later...

Instead lets use [this term](https://www.ebi.ac.uk/ols4/ontologies/ncit/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCIT_C47906):

Output
NCIT:C47906
Anything coming out of a system.